cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -162,7 +162,7 @@ void Run(const std::vector<string>& bag_filenames) {
   }
 
   for (const string& bag_filename : bag_filenames) {
-    if (sigint_triggered) {
+    if (!::ros::ok()) {
```

Comment:
This exactly reverts 

https://github.com/googlecartographer/cartographer_ros/commit/7551d770dc3b569d95b7cde4ba0847201f24238e

@wohe Wolfgang, do you remember why you did this back then?